tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,10 +1,30 @@
+import { Container, Typography } from "@mui/material";
 import React from "react";
 import { useParams } from "react-router-dom";
+import { MAJORS_LIST } from "../constants/careerMajorList";
 
 function MajorPage() {
-  const majorChoice = useParams();
+  const params = useParams();
+  let data = [];
 
-  return <div>{majorChoice.major}</div>;
+  const searchData = (target) => {
+    data = MAJORS_LIST[`"${target}"`];
```

Comment:
Related to the question below, this should be accessible with just `MAJORS_LIST[target]` after adjustments are made, nitpick if also label this as probably just MAJORS because a "list" is no longer an accurate description